internal/distro/rhel84/distro.go

```diff
@@ -1202,6 +1228,113 @@ func newDistro(isCentos bool) distro.Distro {
 		legacy: "i386-pc",
 		uefi:   true,
 	}
+
+	edgeOCIImgTypeX86_64 := imageTypeS2{
+		name:     "rhel-edge-container",
+		filename: "rhel84-container.tar",
+		mimeType: "application/x-tar",
+		packages: []string{
+			"redhat-release", // TODO: is this correct for Edge?
+			"glibc", "glibc-minimal-langpack", "nss-altfiles",
+			"dracut-config-generic", "dracut-network",
+			"basesystem", "bash", "platform-python",
+			"shadow-utils", "chrony", "setup", "shadow-utils",
+			"sudo", "systemd", "coreutils", "util-linux",
+			"curl", "vim-minimal",
+			"rpm", "rpm-ostree", "polkit",
+			"lvm2", "cryptsetup", "pinentry",
+			"e2fsprogs", "dosfstools",
+			"keyutils", "gnupg2",
+			"attr", "xz", "gzip",
+			"firewalld", "iptables",
+			"NetworkManager", "NetworkManager-wifi", "NetworkManager-wwan",
+			"wpa_supplicant",
+			"dnsmasq", "traceroute",
+			"hostname", "iproute", "iputils",
+			"openssh-clients", "procps-ng", "rootfiles",
+			"openssh-server", "passwd",
+			"policycoreutils", "policycoreutils-python-utils",
+			"selinux-policy-targeted", "setools-console",
+			"less", "tar", "rsync",
+			"fwupd", "usbguard",
+			"bash-completion", "tmux",
+			"ima-evm-utils",
+			"audit",
+			"podman", "container-selinux", "skopeo", "criu",
+			"slirp4netns", "fuse-overlayfs",
+			"clevis", "clevis-dracut", "clevis-luks",
+			"greenboot", "greenboot-grub2", "greenboot-rpm-ostree-grub2", "greenboot-reboot", "greenboot-status",
+			// x86 specific
+			"grub2", "grub2-efi-x64", "efibootmgr", "shim-x64", "microcode_ctl",
+			"iwl1000-firmware", "iwl100-firmware", "iwl105-firmware", "iwl135-firmware",
+			"iwl2000-firmware", "iwl2030-firmware", "iwl3160-firmware", "iwl5000-firmware",
+			"iwl5150-firmware", "iwl6000-firmware", "iwl6050-firmware", "iwl7260-firmware",
+		},
+		excludedPackages: []string{
+			"rng-tools",
+			"subscription-manager",
```

Comment:
Looks like `subscription-manager` from [this PR](https://github.com/osbuild/osbuild-composer/pull/1231) already.

Reply:
Yes, I missed that.  Thanks.